hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/ContractFeature.java

```diff
@@ -191,6 +208,67 @@ private MirrorContractResponse verifyContractFromMirror(boolean isDeleted) throw
         return mirrorContract;
     }
 
+    private void verifyContractExecutionResultsById() {
+        List<MirrorContractResult> contractResults = mirrorClient.getContractResultsById(contractId.toString())
+                .getResults();
+
+        assertNotNull(contractResults);
+        assertThat(contractResults).isNotEmpty();
```

Comment:
`isNotEmpty()` also checks for not null.

```suggestion
        assertThat(contractResults).isNotEmpty();
```

Reply:
Updated